Web/starter_project_iconics/styles/singleBlog.css

```diff
@@ -0,0 +1,50 @@
+
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@300&display=swap');
```

Comment:
i am not sure this is the best way to import fonts
I have seen some people do the it in the tailwind configs
might be worth looking into